### DISCUSSION TITLE
FreshRSS API check gray

### DISCUSSION THREAD

**Discussion by Micha10k** [FreshRSS/FreshRSS#6295] [Q&A] [answered]
Hi, 
The last time I installed FreshRSS via CLI on the Raspberry Pi. When I activated the API management, the check mark for the configuration test was green.
![grafik](https://github.com/FreshRSS/FreshRSS/assets/166933182/f0325e7c-94f7-41fe-a673-f6d37568ca61)

Now I have installed FreshRSS via Docker YAML file and the check mark for the configuration test is gray. 
![grafik](https://github.com/FreshRSS/FreshRSS/assets/166933182/634d3285-12fc-4c78-a224-b1aff83f4182)

I can enter the API address in the iOS app Fiery Feeds but no feeds are fetched.
Can anyone help me?

Thanks

**Comment by math-GH** [comment]
The checkmark icon is an UTF8 icon. The design and color if this icon depends on your operating system/device/browser.
In this case: Checkmark means checkmark means PASS :)

**Comment by Micha10k** [comment]
Alright, thanks for your support.
So at least I know that's not the reason why I can't fetch news with Fiery Feeds.

**Comment by math-GH** [comment]
Here is an issue from the past connected with Fiery Feeds and FreshRSS: #2043

**Reply by Micha10k** [reply]
Thanks for the link. I check it out.

**Comment by Alkarex** [comment, marked as answer]
The colour of the check is irrelevant as long as it is a tick and says `pass`. The colour depends on your browser / OS for displaying ✔️
https://emojipedia.org/check-mark
